test/integration_test_runner.rb

```diff
@@ -18,4 +18,21 @@ def test_runner
     assert_path_exist('target/reports/diff/checkstyle/index.html')
     assert_path_exist('target/reports/diff/pmd/index.html')
   end
+
+  def test_single_mode
+    `rake clean`
```

Comment:
For consistency reasons, I would add `rake clean` also for the other test - or better, in the `setup` method of the unit test.